src/one_dragon/base/operation/notify_pool.py

```
@@ -0,0 +1,59 @@
+    """
+
+    def __init__(self) -> None:
+        self.items: list[NotifyPoolItem] = []
+        self.max_images: int = 10
+        self._image_count: int = 0
+
+    def add(self, content: str, image: MatLike | None = None) -> None:
+        """添加一条通知到池中"""
+        if image is not None:
+            self._image_count += 1
+            # 超出图片上限时，移除最旧的图片以释放内存
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)